bindings/python/pyiree/compiler/export.def

```diff
@@ -0,0 +1,17 @@
+;; Copyright 2019 Google LLC
```

Comment:
Nit: these should be 2020 in both files. There's a handy tool that will handle this for you ;-P

Reply:
I update the years manually